src/test/java/uk/gov/register/functional/ApplicationTest.java

```diff
@@ -20,7 +20,7 @@
 
 public class ApplicationTest {
     @ClassRule
-    public static RegisterRule register = new RegisterRule("address");
+    public static RegisterRule register = new RegisterRule();
```

Comment:
no. the whole point of removing the register name is to make the `RegisterRule` more sharable between different tests, to allow us to move from `@Rule` to `@ClassRule` (and from there on to `@Suite` which might allow us to only start a single app server, massively cutting test times. This is as recommended in the dropwizard-testing docs.

Every parameter we add to `RegisterRule` reduces this scope for sharing resources between functional tests and cutting test times.